wix-data/wix-data.service.json

```diff
@@ -1153,6 +1154,7 @@
                 " checking for permissions or without its registered hooks.",
                 "",
                 " > **Notes:** ",
+                " > - To find the itemId of an item, use [`query()`](https://www.wix.com/velo/reference/wix-data/query), or display it in the [CMS](https://support.wix.com/en/article/cms-formerly-content-manager-an-overview): In the CMS UI, open the collection that contains the item you want to get, click **Manage Fields**, and check the ID system field. This will show the ID for all the items in that collection. ",
```

Comment:
Done.

Reply:
The docs for this is very old. We don't do this anymore. You'll see that our style has changed over the years since we evolved from 1 tech writer to about 25.